Plugins/org.secc.PastoralCare/org.secc.PastoralCare.csproj

```diff
@@ -89,6 +86,10 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\Rock\Rock.csproj">
```

Comment:
can we update this reference to use '$(SolutionDir)\Rock\Rock.csproj' that would help with the Azure Pipleline that SECC uses to deploy rock updates to production

Reply:
@JulioCachay the PR looks good, this would be the only update that I would suggest.